cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -407,3 +408,25 @@ func ValidateIgnorePreflightErrors(ignorePreflightErrors []string, skipPreflight
 
 	return ignoreErrors, allErrs.ToAggregate()
 }
+
+// ValidateSocketPath validates format of socket path or url
+func ValidateSocketPath(socket string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	u, err := url.Parse(socket)
+	if err != nil {
+		return append(allErrs, field.Invalid(fldPath, socket, fmt.Sprintf("socket parsing error: %+v", err)))
+	}
+
```

Comment:
extra empty lines can be removed.
having only a `}` on a line is enough separation.

Reply:
ok, will do